client/package.json

```diff
@@ -27,7 +28,8 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "storybook": "start-storybook -p 3600 -s public",
-    "build-storybook": "build-storybook -s public"
+    "build-storybook": "build-storybook -s public",
+    "api": "json-server --watch db.json --port 4001"
```

Comment:
You can remove this script as you don't need it anymore

Reply:
Ah, that makes sense and probably helps solve my deployment error with Heroku :) Thanks!